Microsoft.Toolkit.Uwp.SampleApp/Data/DataGridDataSource.cs

```diff
@@ -30,6 +31,7 @@ public async Task<IEnumerable<DataGridDataItem>> GetDataAsync()
             StorageFile file = await StorageFile.GetFileFromApplicationUriAsync(uri);
             IRandomAccessStreamWithContentType randomStream = await file.OpenReadAsync();
             _items = new ObservableCollection<DataGridDataItem>();
+            IFormatProvider provider = CultureInfo.InvariantCulture.DateTimeFormat;
```

Comment:
QQ: Was there a particular reason we didn't want to use this inline or was it to just bubble it up to the top of the function here?

Reply:
I am not quite sure about the cost of calling `CultureInfo.InvariantCulture.DateTimeFormat`. Consider it is used in a loop, and put it up top seems harmless...

Reply:
@Sergio0694 I'm sure could shed some light here, from my understanding it shouldn't make a difference as it should be optimized by the compiler in either case? Definitely curious now though... Sergio any insights you could shed on this scenario?

Reply:
From my benchmark, there is no difference.


```cs
using System.Globalization;

string t = "01/01/1900";
const int N = 10000000;
IFormatProvider provider = CultureInfo.InvariantCulture.DateTimeFormat;
var watch = new System.Diagnostics.Stopwatch();
watch.Start();
for (int i = 0; i < N; i ++)
{
    DateTimeOffset.Parse(t, provider);
}
watch.Stop();
Console.WriteLine($"Execution Time (top): {watch.ElapsedMilliseconds} ms");

watch.Reset();
watch.Start();
for (int i = 0; i < N; i ++)
{
    DateTimeOffset.Parse(t, CultureInfo.InvariantCulture.DateTimeFormat);
}
watch.Stop();
Console.WriteLine($"Execution Time (inline): {watch.ElapsedMilliseconds} ms");
```

```
Execution Time (top): 3031 ms
Execution Time (inline): 3002 ms
Execution Time (top): 2978 ms
Execution Time (inline): 3010 ms
Execution Time (top): 2991 ms
Execution Time (inline): 3017 ms
Execution Time (top): 2980 ms
Execution Time (inline): 3003 ms
Execution Time (top): 2964 ms
Execution Time (inline): 3031 ms
Execution Time (top): 3038 ms
Execution Time (inline): 3186 ms
Execution Time (top): 2986 ms
Execution Time (inline): 3022 ms
Execution Time (top): 3073 ms
Execution Time (inline): 3014 ms
```

I'll move it inline then :)